### ISSUE TITLE
2 miners

### ISSUE THREAD

**Issue by okxjdeut4567890** [UnamSanctam/SilentCryptoMiner#1844]
If I create 2 miners for a processor and a video card, then in the control panel how will these miners be displayed if both miners work on the same IP and what configuration should be taken by default?

**Comment by UnamSanctam** [comment]
Do you mean how it shows on the web panel?`They will show up as two miners (same computer and IP, different miner IDs) and the GPU miner will use the "Default ethminer" configuration and the CPU miner will use the "Default xmrig" configuration.

**Comment by UnamSanctam** [comment]
Yes, you can change or add configurations on the "Configurations" page with the same format as the "Remote Configuration": https://github.com/UnamSanctam/SilentCryptoMiner/wiki/ETH-Miner#remote-configuration
https://github.com/UnamSanctam/SilentCryptoMiner/wiki/XMR-Miner#remote-configuration
